src/components/atoms/DatasetInfoBox.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Pane, Heading, Text, Badge } from 'evergreen-ui';
+
+type DatasetInfoBoxProps = {
+    title: string;
+    description: string;
+    tags: string[];
+};
+
+const DatasetInfoBox: React.FC<DatasetInfoBoxProps> = ({ title, description, tags }) => {
+    return (
+        <Pane width="100%" height="100%">
+            <Heading size={600}>{title}</Heading>
```

Comment:
Could we add the heading size as an optional prop? And just have the default size be 600

Reply:
I don't think the headings size will differ in any way on different components. This component will be used on the data explorer page and the page where you can customise and add the visualisation to your dashboard, and the styling will be the same. No problem with adding props with default values tho